exporter/datadogexporter/translate_traces.go

```diff
@@ -448,18 +449,18 @@ func getDatadogSpanName(s pdata.Span, datadogTags map[string]string) string {
 	// The spec has changed over time and, depending on the original exporter, IL Name could represented a few different ways
 	// so we try to account for all permutations
 	if ilnOtlp, okOtlp := datadogTags[tracetranslator.TagInstrumentationName]; okOtlp {
-		return strings.ReplaceAll(fmt.Sprintf("%s.%s", ilnOtlp, s.Kind()), "::", "_")
+		return utils.NormalizeTag(fmt.Sprintf("%s.%s", ilnOtlp, s.Kind()))
```

Comment:
It looks weird to me to apply a function called`NormalizeTag` for something that is not a tag name but a span name. Should we change the name of the function or at least add a comment explaining why we apply this to the span name in the code?

Reply:
Yup agreed, i've updated the function here to be more accurate to it's purpose, and renamed the util file as well, see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1373#discussion_r512531520